openwisp_radius/base/fields.py

```diff
@@ -16,7 +16,7 @@ def deconstruct(self):
 
 class FallbackFromDbValueMixin:
     def from_db_value(self, value, expression, connection):
-        if value is None:
+        if value in [None, '']:
             return self.fallback
```

Comment:
can we add logic so that if the value is the same as the default value we set it to `None`?
I thought that's how it worked but it seems to me that's not the case, but I do not remember very well what we did decide a few months ago?